### ISSUE TITLE
removing `impl From` seems to be undetected

### ISSUE THREAD

**Issue by Its-Just-Nans** [obi1kenobi/cargo-semver-checks#1632]
### Is this about an existing lint, or proposing a new one?

New one

### Known issues that might be causing this

- [ ] Is the item that should have been flagged originating from another crate, or from a re-export of such an item? ([#638](https://github.com/obi1kenobi/cargo-semver-checks/issues/638))
- [ ] Is the issue related to a change in the type of a field, function parameter, generic, or trait bound? ([#149](https://github.com/obi1kenobi/cargo-semver-checks/issues/149))
- [ ] If you're proposing a new lint, is it already part of our lint wishlist? ([#5](https://github.com/obi1kenobi/cargo-semver-checks/issues/5))

### Steps to reproduce the bug with the above code

```
impl From<std::string::FromUtf8Error> for MyError {
    fn from(_: std::string::FromUtf8Error) -> Self {
        invalid!("Invalid UTF-8")
    }
}
```

then remove it

### Actual Behaviour

no change detected

### Expected Behaviour

This is a breaking changes no ?

#### Software version

cargo-semver-checks 0.47.0

#### Operating system

- OS: Linux (Ubuntu 24.04)
- Kernel: 6.17.0-20-generic

#### Command-line

```bash
/home/n4n5/.cargo/bin/cargo-semver-checks semver-checks --bugreport 
```

#### cargo version

```
> cargo -V 
cargo 1.94.0 (85eff7c80 2026-01-15)
```

#### Compile time information

- Profile: release
- Target triple: x86_64-unknown-linux-gnu
- Family: unix
- OS: linux
- Architecture: x86_64
- Pointer width: 64
- Endian: little
- CPU features: fxsr,sse,sse2
- Host: x86_64-unknown-linux-gnu

**Comment by obi1kenobi** [comment]
Thanks for creating this. This is spiritually still #149: properly detecting `From` changes requires us to be able to represent the `T` in `From<T>`.

I'm actively working toward that, but it's a ferociously difficult problem. Support in any and all forms appreciated!
